.github/workflows/zcash-params.yml

```diff
@@ -14,98 +14,18 @@ on:
-        with:
-          # list of Docker images to use as base name for tags
-          images: |
-            ${{ env.GAR_BASE }}/${{ env.IMAGE_NAME }}
-          # generate Docker tags based on the following events/attributes
-          tags: |
-            type=schedule
-            type=ref,event=branch
-            type=ref,event=pr
-            type=semver,pattern={{version}}
-            type=semver,pattern={{major}}.{{minor}}
-            type=semver,pattern={{major}}
-            type=sha
-
-      - name: Set up QEMU
-        id: qemu
-        uses: docker/setup-qemu-action@v1
-        with:
-          image: tonistiigi/binfmt:latest
-          platforms: all
-
-      # Setup Docker Buildx to allow use of docker cache layers from GH
-      - name: Set up Docker Buildx
-        id: buildx
-        uses: docker/setup-buildx-action@v1
-
-      # Setup gcloud CLI
-      - name: Authenticate to Google Cloud
-        id: auth
-        uses: google-github-actions/auth@v0.7.1
-        with:
-          workload_identity_provider: 'projects/143793276228/locations/global/workloadIdentityPools/github-actions/providers/github-oidc'
-          service_account: 'github-service-account@zealous-zebra.iam.gserviceaccount.com'
-          token_format: 'access_token'
-
-      - name: Login to Google Artifact Registry
-        uses: docker/login-action@v1.14.1
-        with:
-          registry: us-docker.pkg.dev
-          username: oauth2accesstoken
-          password: ${{ steps.auth.outputs.access_token }}
-
-      # Build and push image to Google Artifact Registry
-      - name: Build & push
-        id: docker_build
-        uses: docker/build-push-action@v2.10.0
-        with:
-          target: builder
-          context: .
-          file: ./docker/zcash-params/Dockerfile
-          # TODO: building crates is taking too long with arm64 and it's timing out on GHA
-          # platforms: |
-          #   linux/amd64
-          #   linux/arm64
-
-          tags: ${{ steps.meta.outputs.tags }}
-          labels: ${{ steps.meta.outputs.labels }}
-          build-args: |
-            SHORT_SHA=${{ env.GITHUB_SHA_SHORT }}
-            ZEBRA_SKIP_IPV6_TESTS="1"
-            SENTRY_DSN=${{ secrets.SENTRY_ENDPOINT }}
-          push: true
-          cache-from: type=registry,ref=${{ env.GAR_BASE }}/${{ env.IMAGE_NAME }}:buildcache
-          cache-to: type=registry,ref=${{ env.GAR_BASE }}/${{ env.IMAGE_NAME }}:buildcache,mode=max
+    # TODO: before merging change uses to ZcashFoundation/zebra/.github/workflows/docker-zebra-build.yml@main
+    uses: ./.github/workflows/docker-zebra-build.yml
+    with:
+      dockerfile_path: ./docker/zcash-params/Dockerfile
+      dockerfile_target: builder
+      image_name: zcash-params
+      network: Mainnet
+      checkpoint_sync: true
+      rust_backtrace: full
+      rust_lib_backtrace: full
+      colorbt_show_hidden: '1'
+      zebra_skip_network_tests: '1'
+      rust_log: debug
```

Comment:
Question: Some of these parameters are unused. Can we document that, to avoid confusion?

```rust
      # only used in tests, this is a build
      network: Mainnet
      checkpoint_sync: true
      zebra_skip_network_tests: '1'
```

Reply:
I removed the unused ones